pkg/hubble/metrics/api/registry_test.go

```diff
@@ -11,8 +11,21 @@ import (
 	"github.com/prometheus/client_golang/prometheus"
 	"github.com/sirupsen/logrus"
 	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
 
 	pb "github.com/cilium/cilium/api/v1/flow"
+	slim_corev1 "github.com/cilium/cilium/pkg/k8s/slim/k8s/api/core/v1"
+	slim_metav1 "github.com/cilium/cilium/pkg/k8s/slim/k8s/apis/meta/v1"
+	monitorAPI "github.com/cilium/cilium/pkg/monitor/api"
+)
+
+var (
+	promRegistry = prometheus.NewRegistry()
+	opts, _      = ParseContextOptions(Options{"sourceContext": "pod", "destinationContext": "pod"})
```

Comment:
Not critical, but would be nice to have similar tests for different set of options, e.g. `sourceContext=workload-name;destinationContext=workload-name;labelsContext=source_pod,source_namespace,destination_pod,destination_namespace` (metrics will be deleted) or `sourceContext=identity;destinationContext=identity;labelsContext=source_pod,destination_pod` (metrics won't be deleted).

Reply:
Good point! New test cases added.